.github/workflows/cron-translations.yml

```diff
@@ -0,0 +1,47 @@
+on:
+  cron:
+    - cron: "19 58 * * *" # every day at 7:58 PM UTC
```

Comment:
That's a strange time. What about just use `0:0 UTC` ?

Reply:
I chose 8pm UTC, as in theory that's what drone is using right now. And I took away 2 minutes because it is using the hosted github runners which fill up from cron users at the `:0` min of each hour.

Similar reasons for the gitignore cron

Reply:
But the "cron" time is around 00:00 AM UTC at the moment.

For example:

https://github.com/go-gitea/gitea/commit/1ef43f9bfebe4529d56e487fd20ce30f34ec7b83

```js
new Date('April 25, 2023 08:07:53 GMT+8').toUTCString()
"Tue, 25 Apr 2023 00:07:53 GMT" 
// "Tue, 25 Apr 2023 00:07:53 GMT" 
```